ext/crt.c

```diff
@@ -263,6 +263,8 @@ static PHP_GINIT_FUNCTION(awscrt) {
     awscrt_globals->log_level = 0;
 }
 
+#define CRT_VERSION "0.1.0"
```

Comment:
This is obviously not the proper version ?

Reply:
yeah, it's not. I was hoping to automate the release process and update this to real version [here](https://github.com/awslabs/aws-crt-php/pull/90/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R38)